pages/03.addons/12.metakube-logging/default.en.md

```diff
@@ -0,0 +1,62 @@
+---
+title: 'Cluster Log Management'
+date: '18-07-2019 13:00'
+taxonomy:
+    tag:
+        - metakube
+        - logging
+        - loki
+        - grafana
+        - addon
+---
+
+The log management Add-On installs a [Loki](https://github.com/grafana/loki/blob/master/docs/README.md) log management stack that integrates into the Grafana instace from the [Monitoring Add-On](../03.metakube-monitoring/default.en.md).
+
+Because of this the Monitoring Add-On is required to be installed before installing the Log Management Add-On.
+
+All logs provided via stdout or sterr by Pods in the cluster will be collected and stored for the configured retention time.
+
+![Grafana Loki Source](grafana_loki_source.png)
+
+## Installed applications
+
+### Loki
+
+Loki is the database where all logs are stored.
+
+* [Documentation](https://github.com/grafana/loki/blob/master/docs/README.md)
+
+### Promtail
+
+Promtail runs as a DaemonSet and collects all logs from all nodes and sends them to Loki.
+
+* [Documentation](https://github.com/grafana/loki/tree/master/docs/clients/promtail)
+
+## Installation
+
+For information on how to install and uninstall an Add-On see [MetaKube Add-Ons](../default.en.md).
+
+## Customization
+
+The following customization options are possible:
+
+### Loki configuration options
+
+| Option | Description | Default value |
+| ------ | ----------- | ------------- |
+| Retention Time | Amount of weeks the logs should be kept | 1 |
+| Persistent Storage | How large should the persistent data volume be | 10Gi |
+| Reject samples older | Log messages older then this value will never be stored | 168h |
```

Comment:
Could this be misunderstood in a way that loki drops data after 168 hours?
Maybe something like "If Promtail sends messages older then this value they will be rejected by Loki"?

Reply:
```suggestion
| Reject samples older | If Promtail sends messages older then this value they will be rejected by Loki | 168h |
```
Yes, your version is much better.